<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Control_Dialog</class>
 <widget class="QDialog" name="Control_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>111</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayoutWave">
    <property name="labelAlignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="1" column="0" colspan="2">
     <widget class="QComboBox" name="comboBoxWaveType"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="lblWaveType">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wave Type</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>111</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayoutCaptureRate">
    <property name="labelAlignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="1" column="0" colspan="2">
     <widget class="QComboBox" name="comboBoxCaptureRate"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="lblCaptRate">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Capture Rate</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>111</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayoutUnits">
    <property name="labelAlignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="1" column="0" colspan="2">
     <widget class="QComboBox" name="comboBoxUnits"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="lblUnits">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Units</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>111</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayoutPulseRate">
    <property name="labelAlignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="1" column="0" colspan="2">
     <widget class="QComboBox" name="comboBoxPulseRate"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="lblPulseRate">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pulse Rate</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>111</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayoutCalc">
    <property name="labelAlignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="1" column="0" colspan="2">
     <widget class="QComboBox" name="comboBoxCalcVar"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="lblCalcVar">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Calculate</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>111</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayoutEMethod">
    <property name="labelAlignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="1" column="0" colspan="2">
     <widget class="QComboBox" name="comboBoxEMethod"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="lblEMethod">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>E Method</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="cdTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>V-Meter Control Parameters</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>111</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayoutGain">
    <property name="labelAlignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="1" column="0" colspan="2">
     <widget class="QComboBox" name="comboBoxAmpGain"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="lblAmpGain">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Amplifier Gain</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>111</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formVoltage">
    <property name="labelAlignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="1" column="0" colspan="2">
     <widget class="QComboBox" name="comboBoxVoltage"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="lblVoltage">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Voltage</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>111</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayoutDataSave">
    <property name="labelAlignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="1" column="0" colspan="2">
     <widget class="QComboBox" name="comboBoxDataSave"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="lblDataSave">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DataSave</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>111</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayoutPicSave">
    <property name="labelAlignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="1" column="0" colspan="2">
     <widget class="QComboBox" name="comboBoxPicSave"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="lblPicSave">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PicSave</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>351</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="formLayoutDensity">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <widget class="QLabel" name="lblDensity">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Density    </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="DensitySpinBox">
      <property name="minimum">
       <number>50</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="horizontalSliderDensity">
      <property name="minimum">
       <number>50</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>351</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="formLayoutDistance">
    <property name="spacing">
     <number>21</number>
    </property>
    <item>
     <widget class="QLabel" name="lblDistance">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Distance       </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="DistanceSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>600</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="horizontalSliderDistance">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>600</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>351</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="formLayoutVelocity">
    <property name="spacing">
     <number>21</number>
    </property>
    <item>
     <widget class="QLabel" name="lblVelocity">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Velocity     </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="VelocitySpinBox">
      <property name="minimum">
       <number>1000</number>
      </property>
      <property name="maximum">
       <number>40000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="horizontalSliderVelocity">
      <property name="minimum">
       <number>1000</number>
      </property>
      <property name="maximum">
       <number>40000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>351</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="formLayoutCycleTime">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <widget class="QLabel" name="lblCycleTime">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CycleTime</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="CycleTimeSpinBox">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>600</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="horizontalSliderCycleTime">
      <property name="maximum">
       <number>600</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>buttonBox</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>layoutWidget_5</zorder>
  <zorder>layoutWidget_6</zorder>
  <zorder>cdTitle</zorder>
  <zorder>layoutWidget_7</zorder>
  <zorder>layoutWidget_8</zorder>
  <zorder>layoutWidget_9</zorder>
  <zorder>layoutWidget_10</zorder>
  <zorder>layoutWidget_11</zorder>
  <zorder>layoutWidget_12</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Control_Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Control_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
